includes/admin/functions.php

```diff
@@ -80,14 +80,16 @@ function amp_admin_get_preview_permalink() {
 }
 
 /**
- * Registers a submenu page to access the AMP template editor panel in the Customizer.
+ * Provides a URL to the customizer.
+ *
+ * @return string
  */
-function amp_add_customizer_link() {
+function amp_get_customizer_url() {
 	$is_legacy = amp_is_legacy();
 
 	/** This filter is documented in includes/settings/class-amp-customizer-design-settings.php */
 	if ( $is_legacy && ! apply_filters( 'amp_customizer_is_enabled', true ) ) {
-		return;
+		return '';
 	}
```

Comment:
This function should also return empty if the mode is Reader, or else it should prevent adding any `$args` other than the `url`.

Reply:
Done in f0d318b52

Reply:
See 141249663 for the actual fix, discussed in our call.